recipe-box-backend/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM maven:3.9-eclipse-temurin-21-alpine
+
+# Create app folder and move local files
+COPY ./pom.xml /recipe-box/
+COPY src /recipe-box/recipe-box-backend/src
+COPY pom.xml /recipe-box/recipe-box-backend/pom.xml
+
+WORKDIR /recipe-box/recipe-box-backend
+
+# Install project dependencies and build production bundle
+RUN mvn package -B -e
```

Comment:
Aici la fel ca in comentariul de la frontend, poti face mai eficient si mai light container-ul daca folosesti doua FROM, unul pentru build stuff si unul pentru containerul final cu strictul necesar.